contracts/argent_account.cairo

```diff
@@ -51,4 +52,5 @@ mod ArgentAccount {
     fn get_contract_address_test() -> felt {
         dummy_syscalls::get_contract_address() 
     }
+
```

Comment:
can you also remove this and the corresponding `use`?

Reply:
yeah sure